### PR TITLE
Various improvements to /attend

### PR DESCRIPTION
**Pull request by strugee**
See commits for details

### CODE REVIEW

File: attend/existing.md [hunk 1]
```diff
@@ -22,11 +26,11 @@ TODO screenshot
 
 Getting here is simple but can be difficult to find on your own. First make sure the spaces list on the far left is expanded (you may have to hover your mouse over the left side to see the arrow):
 
-TODO screenshot
+<img class="align-center" alt="Screenshot of the top-left of the Element client pointing out the arrow directly to the right of the profile picture in the upper-left corner" src="/img/element-sidebar-arrow.webp" />
 
 Then hover over the SeaGL space and click the three dots menu:
 
-TODO screenshot
+<img class="align-center" alt="Screenshot of the SeaGL space item in the Element left pane, with the three dots button at the right of the list item highlighted" src="/img/element-three-dots-menu.webp" />
```
Comment (AndrewKvalheim) [inline attend/existing.md:33]: Might be simpler to tell people to just click on the space twice. On the first click it works like a tab, and on the second click it opens the top level of the space.

No need to expand the side pane or open the dots menu.

Comment (strugee) [reply attend/existing.md:33]: Nice catch, I'll adjust. Rip to all the GIMP editing I did last night lol

Comment (strugee) [reply attend/existing.md:33]: Filed #364 since I want to focus on other higher-priority things - we can circle back and simplify this if there's time

**Comment by strugee** [comment]
Merging per https://github.com/SeaGL/seagl.github.io/pull/363#discussion_r1010007816 so I can get this off my mind

**Pull request merged**